refreshVersions/src/main/kotlin/de/fayard/versions/internal/LatestVersionsRetrieval.kt

```diff
@@ -9,7 +9,11 @@ import kotlinx.coroutines.Dispatchers
 import kotlinx.coroutines.async
 import kotlinx.coroutines.awaitAll
 import kotlinx.coroutines.invoke
+import org.w3c.dom.Element
+import org.w3c.dom.Node
+import org.w3c.dom.NodeList
 import java.net.URL
+import javax.xml.parsers.DocumentBuilderFactory
```

Comment:
I think  `javax.xml.` is broadly available. If you're afraid this might break on some versions of the JVM, we can use Jackson I guess.